HULKs/hulk
# Improved ball search

## Introduced Changes

Add a new note that outputs the most likely ball_position (suggested_ball_position) from a heat map, if the suggested_ball_position has a high enough validity
Change search behavior 
- After loosing the ball the offensive robots will now walk to the suggested_ball_position
- If there is no suggested_ball_position the robot will go back to the old search behavior
Add a heatmap layer in Twix 

## ToDo / Known Issues

- Testing of robot behavior with real robots in a test game
- Maybe some parameter changes

## Ideas for Next Iterations (Not This PR)

- Improve twix heatmap visual design
- Implement more changes to search behavior
- Lower heatmap validity for checked areas
- Set hypothesis for corner kicks, kick ins, etc.
- Better fake data for behavior-simulator

## How to Test

Run ball search Lua test files to see the implemented behavior
Let the Robot kick the ball across the field than hide it to see if he still walks to the last seen ball position
